nvflare/app_opt/pt/file_model_persistor.py

```diff
@@ -204,6 +204,11 @@ def load_model(self, fl_ctx: FLContext) -> ModelLearnable:
         else:
             # if no pretrained model provided, use the generated network weights from APP config
             # note that, if set "determinism" in the config, the init model weights will always be the same
+            self.log_info(
+                fl_ctx,
+                f"Unable to load model from {src_file_name}: Using default data instead.",
```

Comment:
Suggest enhance this message for more clarity:

``f"Both source_ckpt_file_full_name and {AppConstants.CKPT_PRELOAD_PATH} are not provided. Using the default model weights initialized on the persistor side."``